chat-plugins/trivia.js

```diff
@@ -1609,14 +1606,13 @@ const commands = {
 			let questionsLen = questions.length;
 			if (!questionsLen) return this.sendReplyBox(`No questions have been submitted for ${isWL ? "Weakest Link" : "Trivia"} yet.`);
 
-			let categories = Object.keys(CATEGORIES);
 			let lastCategoryIdx = 0;
 			buffer += "<tr><th>Category</th><th>Question Count</th></tr>";
-			for (let i = 0; i < categories.length; i++) {
-				if (categories[i] === 'random') continue;
-				let tally = findEndOfCategory(categories[i], false) - lastCategoryIdx;
+			for (const category of Object.keys(CATEGORIES)) {
```

Comment:
Honestly, `const category in CATEGORIES`...

`Object.keys` is superfluous if you're just going to iterate.